### DISCUSSION TITLE
Using as on a Tab.Panel with Custom Component

### DISCUSSION THREAD

**Discussion by dapacreative** [tailwindlabs/headlessui#2244] [Help]
When using a custom component on a tab.panel. Is there a way to pass props to have the id and aria-labeledby index match the tabs? I believe that index is auto generated.

**Comment by dapacreative** [comment]
Ultimately I just want to be able to set the tabIndex of the Tab.Panel to -1 but that is currently a bug in the React version.

**Reply by RobinMalfait** [reply]
This should work in the latest version that was released a couple of days ago.
